sale_order_invoicing_finished_task/__manifest__.py

```diff
@@ -5,7 +5,8 @@
 
 {
     "name": "Sale Order Invoicing Finished Task",
-    "summary": "Control invoice order lines if his task has been finished",
+    "summary": "Control invoice order lines if its related task has been "
```

Comment:
Sorry, I told you bad. It's `their related`, not `its related`. Please change this and squash directly the commits.